index.html

```diff
@@ -1,11 +1,41 @@
-Ki Vera Luna Pardillo
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
It's good practice to have a `title` element in your `head`. This helps with Search Engine Optimization and also updates the title of the tab in the browser: [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title)

Eg:

`<title>Ki's Homepage</title>`